docs/general/how-ra-works.md

```diff
@@ -0,0 +1,45 @@
+# How RetroAchievements Works
+
+RetroAchievements (RA) provides users the ability to earn achievements in retro games from an RA achievement set.  It awards achievements by comparing a game’s memory, henceforth referred to as RAM, to achievement code written by an RA developer.  Achievement code, also known as logic, is a list of memory conditions chosen by the developer, that when all are simultaneously true on a single frame, will award an achievement.
+
+## What Is an Achievement Set?
+
+An achievement set is the compilation of individual achievements, Rich Presence and leaderboards for a particular game.  Achievements are comprised of achievement logic, a title, a description, a point value and a badge.  Additionally, a set must contain a Rich Presence script which provides site users information on where active players are in a game.  Sets may have leaderboards that track how well players perform in certain things in a game such as how quickly they can beat stages, how many points they can score and many other things.  RA users interface with all these components and subcomponents.  Each one is a vital part of the player experience.
```

Comment:
Recommend changing "Achievements are comprised of" to "Achievements consist of".

"X is comprised of" is a common grammatical error.

Reply:
Updated.